.github/workflows/antithesis-test-3.5.yml

```diff
@@ -0,0 +1,110 @@
+---
+name: Build and trigger Antithesis exploration
+
+on:
+  # Disabled as discussed in https://github.com/etcd-io/etcd/pull/19750#issuecomment-2809840402
+  # pull_request:
+  #   branches: [main]
+  # schedule:
+  #   - cron: "0 0 * * *" # run every day at midnight
+  workflow_dispatch:
+    inputs:
```

Comment:
I think having multiple files is too much to handle. Maybe we could have 1 file, and use workflow `inputs` to provide `etcd_version` parameter, or `etcd_ref` to allow selecting arbitrary branch.

https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputs